addons/payment/static/src/js/payment_token.js

```diff
@@ -0,0 +1,252 @@
+odoo.define('payment.payment_tokens', function (require){
+"use strict";
+
+var ajax = require('web.ajax');
+var core = require('web.core');
+var Dialog = require("web.Dialog");
+
+$(document).ready(function (){
```

Comment:
Wait is this module a web client module or a website module? Because if it's the former, this makes no sense whatsoever.